src/file_organizer/web/settings_service.py

```
@@ -0,0 +1,326 @@
+            apply_web_settings_payload(ws, raw)
+            return sanitize_web_settings(ws)
+        except Exception as exc:
+            logger.warning("Failed to load settings from {}: {}", self.settings_file, exc)
+            return WebSettings()
+
+    def save(self, ws: WebSettings) -> None:
+        """Persist settings to disk."""
+        try:
+            self.settings_dir.mkdir(parents=True, exist_ok=True)
+            atomic_write_text(self.settings_file, json.dumps(asdict(ws), indent=2) + "\n")
+        except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)